Week2/homework/array.js

```diff
@@ -3,7 +3,7 @@ let animals = [];
 
 
 //6.2 Write a console.log statement that explains in words what you think the value of the array is.
-console.log('the content of my array will be about animals');
+console.log('the value of my array will be zero');
```

Comment:
Ah, almost correct! In your comment you said it correct, the value of `my array` is an `empty` array. So it is not zero, but an empty array:)

Reply:
done!